gunicorn.Dockerfile

```diff
@@ -4,15 +4,20 @@ ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get update \
 &&  apt-get upgrade -y \
-&&  apt-get install -y --no-install-recommends gunicorn \
+&&  apt-get install -y --no-install-recommends gunicorn python3-psycopg2 \
 &&  apt-get clean \
 &&  rm -rf /var/cache/apt/archives/* /var/lib/apt/lists/*
 
 WORKDIR /srv/gunicorn/
 
-COPY run .
+COPY ./docker/run .
 RUN chmod +x run
 
-EXPOSE 80
+EXPOSE 8080
```

Comment:
I don't think you need this `EXPOSE` instruction

Reply:
We need it for the --bind option in the run file (gunicorn command)